src/ShapeCrawler/Presentations/ISlide.cs

```diff
@@ -66,6 +71,11 @@ public interface ISlide
     /// </summary>
     ITable TableWithName(string table);
     
+    /// <summary>
+    ///     Adds a notes slide if there isn't already one.
+    /// </summary>
+    void AddNotesIfEmpty();
```

Comment:
How about renaming and slightly changing the method signature to `AddNotes(string notes)`?

As mentioned earlier, the idea is to make the use of the high-level library's API closer to the PowerPoint user experience.

Reply:
Ok, I changed to take enumerable of strings, as would be better to add all the paragraphs at once if we're adding many.